.nais/send-varsler-job-prod.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
     team: team-esyfo
 spec:
   image: {{ image }}
-  schedule: "5 10 * * *"
+  schedule: "5 11 * * *"
```

Comment:
Jeg tror du må sette den til 12, fordi den kjører to timer senere

Reply:
Done